integration_tests/features/support/steps.js

```diff
@@ -250,12 +250,18 @@ Given(
     const nodeA = this.getNode(nodeNameA);
     const nodeB = this.getNode(nodeNameB);
     nodeA.setPeerSeeds([nodeB.peerAddress()]);
+    console.log("Stopping node");
     await this.stopNode(nodeNameA);
+    console.log("Starting node");
     await this.startNode(nodeNameA);
     await waitFor(
       async () => {
-        let node_a_result = (await nodeA.get_node_state()) === "LISTENING";
-        let node_b_result = (await nodeB.get_node_state()) === "LISTENING";
+        let nodeAState = await nodeA.get_node_state();
+        console.log(`${nodeNameA} state: ${nodeAState}`);
+        let nodeBState = await nodeB.get_node_state();
+        console.log(`${nodeNameB} state: ${nodeBState}`);
+        let node_a_result = nodeAState === "LISTENING";
+        let node_b_result = nodeAState === "LISTENING";
         return node_a_result && node_b_result;
```

Comment:
I suggest not having these checks in here. It is already tested for in `baseNodeProcess.js/start -> baseNodeProcess.js/run` and includes other variants of possible console output as well.

I found that these additional checks let this step time out, possibly due to `await this.client.GetTipInfo().sendMessage({});` not returning directly after base node startup.

Reply:
Thanks I see that now